ios/Classes/AblyFlutterPlugin.m

```diff
@@ -145,7 +203,10 @@ -(instancetype)initWithChannel:(FlutterMethodChannel *const)channel streamsChann
         AblyPlatformMethod_publish: _publishRestMessage,
         AblyPlatformMethod_createRealtimeWithOptions: _createRealtimeWithOptions,
         AblyPlatformMethod_connectRealtime: _connectRealtime,
-        AblyPlatformMethod_closeRealtime: _closeRealtime
+        AblyPlatformMethod_closeRealtime: _closeRealtime,
+        AblyPlatformMethod_attachRealtimeChannel: _attachRealtimeChannel,
+        AblyPlatformMethod_detachRealtimeChannel: _detachRealtimeChannel,
+        AblyPlatformMethod_setRealtimeChannelOptions: _setRealtimeChannelOptions
```

Comment:
For the benefit of future diff clarity can you please always add trailing `,` to the end of the last item in dictionaries and arrays.

Reply:
Addressed @ 2e30875812428b10eee203ca12699f9c343eba8a